packages/transactions/tests/builder.test.ts

```diff
@@ -2359,3 +2370,279 @@ test.each([
 
   expect(bytesToHex(transaction.serialize())).toEqual(txBytes);
 });
+
+describe('multi-sig', () => {
```

Comment:
I would add a couple tests where you make a valid transaction, flip a bit or two, and expect `verifyOrigin()` to fail